templates/cli/app/services/client.php.twig

```diff
@@ -49,4 +72,22 @@ $cli
        Console::log('Server Version : {{ spec.version | url_encode }}');
     });
 
+
+$cli
+    ->task('help')
+    ->action(function() use ($parser) {
+        Console::log("\e[0;31;m {{ language.params.logo | raw }} \e[0m") ;
+        Console::log("\nUsage : {{ language.params.executableName }} client [COMMAND]\n");
+        Console::log("Commands :");
+        $commands = [
+            "setEndpoint" => "Set your server endpoint.",
```

Comment:
This is the data you wanted me to add to the spec, right?

Reply:
Yeap. It would be great if we could add `client` and `init` as services or under some other key in the spec. 

so I can iterate over the methods in the twig templates like so
```
{% for method in service.methods %}
 I would want methods to contain [setEndpoint, setProject, setKey, setLocale] for client and each method should have the required parameter and their description just like how we have for the other services. 
{% endfor %}
```